aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -84,25 +91,28 @@ The example below sets two environment variables. `ASPNETCORE_ENVIRONMENT` will
 </aspNetCore>
 ```
 
+> [!WARNING]
+> Only set the `ASPNETCORE_ENVIRONMENT` envirnonment variable to `Development` on staging and testing servers that aren't accessible to untrusted networks, such as the Internet.
+
 ## app_offline.htm
 
-If you place a file with the name *app_offline.htm* at the root of a web application directory, the ASP.NET Core Module will attempt to gracefully shutdown the app and stop processing incoming requests. If the app is still running after `shutdownTimeLimit` number of seconds, the ASP.NET Core Module will kill the running process.
+If a file with the name *app_offline.htm* is detected in the root directory of an app, the ASP.NET Core Module attempts to gracefully shutdown the app and stop processing incoming requests. If the app is still running after `shutdownTimeLimit` number of seconds, the ASP.NET Core Module kills the running process.
```

Comment:
~~If the app is still running after `shutdownTimeLimit` number of seconds~~
If the app is still running after the number of seconds defined in `shutdownTimeLimit`